presto-main/src/main/java/com/facebook/presto/resourcemanager/ResourceManagerClusterStateProvider.java

```diff
@@ -198,21 +206,52 @@ else if (!info.getState().isDone()) {
                 .map(nodeStatus -> nodeStatus.getNodeStatus().getMemoryInfo())
                 .collect(toImmutableList());
 
-        Map<MemoryPoolId, Long> counts = nodeQueryStates.values().stream()
-                .map(CoordinatorQueriesState::getActiveQueries)
-                .flatMap(Collection::stream)
-                .collect(groupingBy(query -> query.getBasicQueryInfo().getMemoryPool(), counting()));
-        // Add defaults when queries are not running
-        counts.putIfAbsent(GENERAL_POOL, 0L);
+        int generalPoolCounts = 0;
```

Comment:
may be call it `queriesAssignedtoGeneralPool` to refrect what exactly this count is for.